sympy/printing/mathml.py

```diff
@@ -464,6 +464,30 @@ class MathMLPresentationPrinter(MathMLPrinterBase):
     """
     printmethod = "_mathml_presentation"
 
+    def __init__(self, settings=None):
+        MathMLPrinterBase.__init__(self, settings)
+
+        _default_settings = {
+            "fold_frac_powers": False,
+            "fold_func_brackets": False,
+            "fold_short_frac": None,
+            "inv_trig_style": "abbreviated",
+            "ln_notation": False,
+            "long_frac_ratio": None,
+            "mat_delim": "[",
+            "mat_symbol_style": "plain",
+            "mul_symbol": None,
+            "root_notation": True,
+            "symbol_names": {},
+            "order": None
+        }
+
+        self._settings = _default_settings
+
+        if settings is not None:
+            for setting in settings:
```

Comment:
How about using a copy of `settings`?

Reply:
But then, `settings` need not have all the items, so if we do a copy, the rest of the items will be lost.
Did you mean something like
`self._settings.update(settings)` 
This would retain the default items, which are not present in `settings`